docs/Options/GenerateOptions.md

```diff
@@ -19,3 +19,11 @@ Options are optional and case-insensitive
     * Validate the profile, check to see if known [contradictions](../../generator/docs/Contradictions.md) exist, see [Profile validation](../../generator/docs/ProfileValidation.md) for more details
 * `--trace-constraints`
    * When generating data emit a `<output path>.trace.json` file which will contain details of which rules and constraints caused the generator to emit each data point.
+
+Standard monitoring shows velocity and row data, alternatives are:
```

Comment:
```suggestion
By default the generator will report how much data has been generated over time, the other options are below:
```

Reply:
Can you also change line 13 to show that the reductive walker is now the default - rather than the cartesian product walker.